public/locales/pt/common.json

```
@@ -1,7 +1,7 @@
 {
-  "title": "O modo mais fácil de criar um",
+  "title": "A forma mais fácil de criar um",
   "readme": "README",
-  "description": "O nosso simples editor permite que você facilmente adicione e personalize todas as seções que você precisa para o README do seu projeto",
+  "description": "O nosso simples editor permite adicionar e personalizar facilmente todas as seções que você precisa para o README do seu projeto",
```

I liked how the changes simplifies the reading! My suggestion to go further on simplifying it would be:

```suggestion
  "description": "O nosso simples editor permite adicionar e personalizar facilmente todas as seções do README de seu projeto",
```

---

I though about simplifying this part too, but I felt that deviates too much from the original English string:

> Our simple editor allows you to quickly add and customize all the sections you need for your project's readme